src/sequence_batch_scheduler.cc

```diff
@@ -976,8 +1107,49 @@ SequenceBatchScheduler::ReaperThread(const int nice)
   LOG_VERBOSE(1) << "Stopping sequence-batch reaper thread...";
 }
 
+void
+SequenceBatchScheduler::CleanUpThread(const int nice)
+{
+#ifndef _WIN32
+  if (setpriority(PRIO_PROCESS, syscall(SYS_gettid), nice) == 0) {
+    LOG_VERBOSE(1) << "Starting sequence-batch clean-up thread at nice " << nice
+                   << "...";
+  } else {
+    LOG_VERBOSE(1) << "Starting sequence-batch clean-up thread at default nice "
+                      "(requested nice "
+                   << nice << " failed)...";
+  }
+#else
+  LOG_VERBOSE(1)
+      << "Starting sequence-batch clean-up thread at default nice...";
+#endif
```

Comment:
Can this be a helper re-used in both thread functions?

Reply:
Looks like `BatcherThread` could reuse it too

Reply:
Nice spotting! Updated on commit: [Group set thread priority into a function](https://github.com/triton-inference-server/core/pull/207/commits/db77e3759b34df84bcc9239784995a33ac81f4c3)